src/Modules/Visualization/ShowField.cc

```diff
@@ -240,81 +241,45 @@ GeometryHandle ShowFieldModule::buildGeometryObject(
   const int dim = field->vmesh()->dimensionality();
   if (showEdges && dim < 1) { showEdges = false; }
   if (showFaces && dim < 2) { showFaces = false; }
+  //adjust color map to scalar data if there is one.
```

Comment:
I finally get it... yes, i'll take it out and check that it works as intended with/without rescale.